python/ray/job_config.py

```diff
@@ -13,28 +13,27 @@ class JobConfig:
         code_search_path (list): A list of directories or jar files that
             specify the search path for user code. This will be used as
             `CLASSPATH` in Java and `PYTHONPATH` in Python.
+        runtime_env (dict): A dict used to specify runtime environment of
+            the job. There are three important fields.
+            - `working_dir (str)`: A string of working directory
+            - `working_dir_uri (str)`: A string of uri of the package stored.
```

Comment:
Same as `working_dir`, but a URI referencing a stored archive instead of a local path. Takes precedence over `working_dir`.